k8s/liara/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
+#    Copyright 2017 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+FROM legion/base-python-image
+
+ARG pip_extra_index_params=""
+ARG pip_legion_version_string=""
+
+RUN pip install $pip_extra_index_params legion$pip_legion_version_string
+
+ENV TINI_VERSION v0.16.1
```

Comment:
This functionality is now supported by docker itself. There is not need for TINI